cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```diff
@@ -85,6 +84,21 @@ spec:
               mountPath: /mount-cache-dir
             - name: ceph-csi-config
               mountPath: /etc/ceph-csi-config/
+        - name: liveness-prometheus
+          image: {{ .CSIPluginImage }}
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /csi
+          args:
+            - "--type=liveness"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--port=8080"
+            - "--livenessendpoint=/metrics"
+            - "--poll-time=60"
```

Comment:
don't we need to  add a comment saying poll-time is in seconds (it might help users)